### PR TITLE
Filters - Example of Date Dropdown

### PR DESCRIPTION
**Pull request by Kidglove57**
Add Screenshots re Date Dropdown to Docs

### CODE REVIEW

File: docs/reports-filters/filters.md [hunk 1]
```diff
@@ -53,9 +53,9 @@ But what if we need to drill down further and calculate just the costs were for
 
 In the above Example we used the **Date is greater than** and **Date is less than** filters to illustrate the fine tuning that is possible. However, in this example of a simple calendar year it would have been quicker simply to click on the down arrow below **Date** and select as follows:
 
- ![](/img/filtering/dates.png)
+ ![](/img/filtering/Dates1.png)
```
Comment (j-f1) [inline docs/reports-filters/filters.md:56]: It looks like both of the images here are Retina — can you rename them to e.g. `Dates1@2x.png` and `Dates2@2x.png`?

Also— how did you take the screenshots? It looks like they are actually a JPEG file with a `.png` extension. Can you either retake them as PNGs (preferred) (that should be the default format for the macOS built-in screenshot tool, if you’re using a different screenshot tool it should have a setting to change the format) or rename the file extension to `jpeg`?

Comment (Kidglove57) [reply docs/reports-filters/filters.md]: Sorry about this @j-f1 .

I have redone the images in the right .png format and renamed as you recommend.
But now I have all this spellcheck failure which I don't understand. I'm finding GitHub desktop quite unintuitive.

Comment (j-f1) [reply docs/reports-filters/filters.md]: Thanks for updating! The spell check is a new thing we’re testing out. I think the issue is that your branch was out of date but I’ve updated it. The spell check should succeed now.

The spell checker is a new step that we run when you push some code up to help avoid spelling issues in the future (alongside the existing step that builds a preview version of the website).

Comment (Kidglove57) [reply docs/reports-filters/filters.md]: Thanks for explaining. I did get a bit lost trying to update in the desktop app.
Ready for me to merge?

**Review by j-f1 — APPROVED** [review]
Looks good — thanks for adding these screenshots!

File: docs/reports-filters/filters.md (hunk 1 above)
Comment (j-f1) [reply docs/reports-filters/filters.md]: Yep!

Comment (j-f1) [reply docs/reports-filters/filters.md]: There’s an “update branch” button on the pull request page that you can click to quickly update your branch, and then you can click Pull in GitHub Desktop to pull down the newly created merge commit that updated your branch.

**Pull request merged**